src/sortStudents.ts

```diff
@@ -1,16 +1,95 @@
+// import { type } from "os";
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(students: Student[], sortBy: SortType,
+  order: SortOrder): Student[] {
```

Comment:
You have to choose to write function parameters inline or from new lines and use this strategy for EVERY parameter
E-g: `function sortStudents(students: Student[], sortBy: SortType,order: SortOrder): Student[] {` or 
```suggestion
export function sortStudents(
  students: Student[],
  sortBy: SortType,
  order: SortOrder
): Student[] {
```